Makefile

```diff
@@ -637,7 +637,7 @@ generate-package-secret: ## Generate the default package values secret. Usage: m
 	@if [ $(PACKAGE) == 'pinniped' ]; then \
 	  ./addons/pinniped/tanzu-auth-controller-manager/hack/generate-package-secret.sh -v tkr=${tkr} -v infrastructure_provider=${iaas} ;\
 	elif [ $(PACKAGE) == 'capabilities' ]; then \
-	  ./pkg/v1/sdk/capabilities/hack/generate-package-secret.sh -v tkr=${tkr} ;\
+	  ./pkg/v1/sdk/capabilities/hack/generate-package-secret.sh -v tkr=${tkr} --data-value-yaml 'rbac.podSecurityPolicyNames=[${psp}]';\
```

Comment:
Any way to make this generic so you could use `--data-value-yaml` to override other values too?

Reply:
I tried passing a list of maps(for tolerations) using `--data-value-yaml` and I think it's better to pass the values file as input rather than CLI inputs. For now, I will go with this approach, please do let me know if you think otherwise.